website/content/en/docs/building-operators/golang/operator-scope.md

```diff
@@ -44,7 +44,7 @@ mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 
 ### Watching resources in specific Namespaces
 
-To restrict the scope of the [Manager's][ctrl-manager] cache to a specific Namespace set the `Cache.DefaultNamespaces'
+To restrict the scope of the [Manager's][ctrl-manager] cache to a specific Namespace set the `Cache.DefaultNamespaces`
```

Comment:
I would add a comma after "specific Namespace". and drop "the" before `Cache.DefautlNamespaces` so that it reads, "To restrict the scope of the Manager's cache to a specific Namespace, set Cache.DefaultNamespaces"

Reply:
Thanks @OchiengEd 
Updated per your suggestion